### ISSUE TITLE
Scroll freezes on rapid slide upward... 

### ISSUE THREAD

**Issue by bvahedy** [AndBible/and-bible#278]
If you try to scroll up fast when the page gets to the end of a chapter the page goes white  and scroll freezes.
You can however scroll down a page and then the issue goes away.
I think this may have something to do with continuous verse across chapter routine.

**Comment by tuomas2** [comment]
Have you still experienced this with latest builds? @bvahedy

**Comment by bvahedy** [comment]
Yes this is still happening in build 333. If you scroll up fast you end up
with a white screen, I waited a minute and nothing happened, but if you
slide down slowly to go up a verse the screen starts filling with verses.

**Comment by timbze** [comment]
Outdated
